### PR TITLE
Footer links edit

### PR DESCRIPTION
**Pull request by MAlngaawy**
1 -  make Links open in new tab 
2 - add a unique key to the li element 
3 - remove the unused underline destructure

### CODE REVIEW

File: frontend/components/TeamSection/index.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const TeamGroupSection = ({ data }) => {
           gradientColor={HEADING_OPTIONS.GRADIENT_COLOR.GREEN}
         />
       </Heading>
-      <div className="grid md:grid-cols-2 sm:grid-cols-1 md:px-48 gap-5">
+      <div className="grid md:grid-cols-2 sm:grid-cols-1 lg:px-48 gap-5">
```
Comment (Tramsi) [inline frontend/components/TeamSection/index.jsx:30]: Merge the latest changes of develop branch

File: frontend/components/footer/index.jsx [hunk 2]
```diff
@@ -40,10 +40,10 @@ const Footer = ({ data }) => {
             <h3 className="text-base mb-6 font-bold">Links</h3>
             <ul className="flex flex-wrap">
               {links.map(link => {
-                const { url, underline, name } = link;
+                const { url, name, id } = link;
                 return (
-                  <li className="mb-4 leading-7 font-bold underline w-1/2">
-                    <a href={url}>{name}</a>
+                  <li key={id} className="mb-4 leading-7 font-bold underline w-1/2">
+                    <a href={url} target="_blank">{name}</a>
```
Comment (Tramsi) [inline frontend/components/footer/index.jsx:46]: I suggest adding the rel="noopener noreferrer" to the anchor tag

File: frontend/components/TeamSection/index.jsx (hunk 1 above)
Comment (MAlngaawy) [reply frontend/components/TeamSection/index.jsx]: I merge the last changes of develop branch
same changes no problem here!

File: frontend/components/footer/index.jsx (hunk 2 above)
Comment (MAlngaawy) [reply frontend/components/footer/index.jsx]: Done

**Pull request merged**